ansible/vars/icds/icds_public.yml

```diff
@@ -0,0 +1,197 @@
+SITE_HOST: 'icds-cas.gov.in'
+NO_WWW_SITE_HOST: 'icds-cas.gov.in'
+deploy_env: icds
+
+riakcs_s3_access_key: "{{ secrets.RIAKCS_S3_ACCESS_KEY }}"
+riakcs_s3_secret_key: "{{ secrets.RIAKCS_S3_SECRET_KEY }}"
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: False
+  cchq_http: True
+  cchq_http_redirect: True
+  cchq_http_j2me: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: True
+  dhis: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech1: False
+  motech2: False
+  riakcs: True
+  cas_ssl: False
+
+proxy_type: 'nginx'
+primary_ssl_env: "icds"
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+fake_ssl_cert: no
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_cluster_name: 'prodhqes-1.x'
+
+supervisor_http_enabled: True
+supervisor_http_username: "{{ secrets.SUPERVISOR_HTTP_USERNAME }}"
+supervisor_http_password: "{{ secrets.SUPERVISOR_HTTP_PASSWORD }}"
+
+keystore_file: '../config/DimagiKeyStore'
+
+backup_blobdb: False
+backup_postgres: True
+backup_es: True
+postgres_s3: False
+
+postgres_users: "{{ secrets.POSTGRES_USERS }}"
+
+postgres_config:
+  max_connections: 300
+  pgb_max_connections: 400
+  pgb_default_pool: 290
+  pgb_reserve_pool: 5
+  pgb_pool_timeout: 1
+  pgb_pool_mode: transaction
+
+formplayer_db_name: formplayer
+
+postgresql_dbs:
```

Comment:
@snopoke had to change the hosts from `pg0` to `{{ groups.postgresql.0 }}` to make the playbook run since it does a check by ip. also, it seems to have created all databases on both machines. was that expected? still need to run the migrations but will do those tomorrow so I cant confirm if its working yet.